CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.0
+  - Feat: ECS compatiblity [#9](https://github.com/logstash-plugins/logstash-filter-syslog_pri/pull/9) 
```

Comment:
```suggestion
  - Feat: ECS compatibility [#9](https://github.com/logstash-plugins/logstash-filter-syslog_pri/pull/9) 
```

Reply:
Please fix the spelling of "compatibility" in the PR title and changelog (because we link to them from release notes).  :-)